tox.ini

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length=120
 application_import_names=pysite
-ignore=P102
+ignore=P102,B311,W503,E226,S311
```

Comment:
this seems to disable 
- **W503**: line break before binary operator
- **E226**: missing whitespace around arithmetic operator
and two others for which I can't find what they mean?. Where are these actually required? I feel like disabling this globally might be a bit overkill, unless you absolutely have to do it this way everywhere

Reply:
These are the ones we have disabled on the bot, which is why I pulled them over quite readily. 

- **W503** goes against PEP8, so that one is bullshit. PEP8 recommends that line breaks _should_ come before arithmetic operators, because that has much better readability.

- **B311** completely forbids importing the `random` module, because it considers it insecure for cryptographical purposes. While I don't actually use `random` in this PR, this is still a completely unreasonable linting rule that we don't need.

- **S311** - uhhhh I'm not really sure what this one does but I think maybe it's the same as B311. We had to disable this one during code jam because it fucked with so many people.

- **E226** is bad because PEP8 has provisions for sometimes not having whitespace around operators. See the following:
![image](https://user-images.githubusercontent.com/2098517/39069948-cc96805c-44e1-11e8-88ab-3ff4227eeeae.png)

Reply:
Thanks for the explanation. All good then 👍